Dockerfile

```diff
@@ -0,0 +1,46 @@
+# frost
+
+FROM python:3.8-slim-buster
+
+ENV PYTHONPATH $PYTHONPATH:/app
+ENV PYTHONUNBUFFERED 1
+
+RUN groupadd --gid 10001 app && \
+    useradd --uid 10001 --gid 10001 --shell /usr/sbin/nologin app
+RUN install -o app -g app -d /var/run/depobs /var/log/depobs
+
+# git for herokuadmintools
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && \
+        apt-get upgrade -y && \
+        apt-get install --no-install-recommends -y \
+            ca-certificates \
+            curl \
+            git \
+            jq
+
+WORKDIR /app
+
+COPY requirements.txt .
+RUN pip install --upgrade --no-cache-dir -r requirements.txt
+
+COPY CODE_OF_CONDUCT.md .
+COPY LICENSE .
+COPY Makefile .
+COPY README.md .
+COPY cache.py .
+COPY conftest.py .
+COPY custom_config.py .
+COPY exemptions.py .
+COPY meta_test_cache.py .
+COPY regressions.py .
+COPY service_report_generator.py .
+COPY severity.py .
+COPY bin bin
+COPY aws aws
+COPY gcp gcp
+COPY gsuite gsuite
+COPY heroku heroku
+COPY pagerduty pagerduty
```

Comment:
Why all of these `COPY` instructions rather than a `COPY * .`?

Reply:
Feel like this could easily get out of sync with new additions and could be kind of a pain to determine if everything we need is listed here.

Reply:
yeah that + a `.dockerignore` is probably cleaner I'll update it